UnitySDK/Assets/ML-Agents/Scripts/Academy.cs

```diff
@@ -594,6 +614,9 @@ protected virtual void OnDestroy()
 
             // Signal to listeners that the academy is being destroyed now
             DestroyAction();
+
+            m_Timer.SaveJsonTimers();
```

Comment:
This does not currently trigger when learn.py closes the environment. Need to debug that further, but will do so in a separate PR.

Reply:
Fixed in another PR.